test/active_support_test_case_helper.rb

```diff
@@ -56,7 +56,7 @@ def reset_setting_cache
   alias_method :assert_include,     :assert_includes
   alias_method :assert_not_include, :assert_not_includes
   class <<self
-    alias_method :test, :it
+    # alias_method :test, :it
```

Comment:
I ran them locally, and I'm surprised to see this pass https://github.com/theforeman/foreman/blob/develop/test/functional/application_controller_subclass_test.rb All good then!